src/lib/components/pages/anime_info/error.svelte

```diff
@@ -53,7 +53,7 @@
                 {@html format_kokoro_color(`Uh-oh looks like our kokoro-chan is working really hard for the past few days and now has fall asleep. You can wait for her to wake up by looking at the status page, or come say hi to other fellow kokoro-chan's worksippers! ah- also let's wish her sweat dreams!`)}
             </context>
             <a
-                href="/explore"
+                href="../explore"
```

Comment:
Relative linking is bad idea

Reply:
Indeed, thats why I made this PR: https://github.com/tokitou-san/CoreProject-V3-UI/pull/683